UnityProject/Assets/Scripts/Messages/Client/Interaction/RequestInteractMessage.cs

```diff
@@ -256,7 +258,8 @@ public static void Send<T>(T interaction, IBaseInteractable<T> interactableCompo
 		{
 			ComponentType = interactableComponent.GetType(),
 			InteractionType = typeof(T),
-			ProcessorObject = comp.GetComponent<NetworkIdentity>().netId
+			ProcessorObject = comp.GetComponent<NetworkIdentity>().netId,
+			Intent = interaction.Intent
 		};
 		if (typeof(T) == typeof(PositionalHandApply))
 		{
```

Comment:
an attempt to melee hit yourself is a PositionalHandApply, so damage zone should go here instead of HandApply
```suggestion
		{
			var casted = interaction as PositionalHandApply;
			msg.TargetBodyPart = casted.TargetBodyPart;
```
otherwise it defaults to Head for some reason